### PR TITLE
CLI code changes for Boundaryscaling feature

### PR DESCRIPTION
**Pull request by ravindrakreddy**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by ravindrakreddy** [comment]
@QingqingYuan please review

File: src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py [hunk 1]
```diff
@@ -158,9 +158,11 @@ def _server_update_custom_func(instance,
     module = import_module(server_module_path.replace('server', 'server_update_parameters'))
     ServerUpdateParameters = getattr(module, 'ServerUpdateParameters')
 
-    if capacity is not None:
-        instance.sku.name = _get_sku_name(instance.sku.tier, instance.sku.family, capacity)
-        instance.sku.capacity = capacity
+    if sku_name is not None:
```
Comment (troydai) [inline src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py]: Why not just 
`if sku_name:`?

File: src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py [hunk 2]
```diff
@@ -90,6 +90,7 @@ def _test_server_mgmt(self, database_engine, resource_group_1, resource_group_2)
         new_cu = 4
         family = 'Gen5'
         skuname = '{}_{}_{}'.format("GP", family, old_cu)
+        newskuname = '{}_{}_{}'.format("GP", family, new_cu)
```
Comment (troydai) [inline src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py]: Why not `'GP_{}_{}'.format(family, new_cu)`?

File: src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py (hunk 1 above)
Comment (ravindrakreddy) [reply src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py]: sku has additional properties like capacity. Thats why it is specified sku.name

Comment (troydai) [reply src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py]: No. My question is why not just test if the `sku_name` is `None` simply using `if sku_name`

Comment (troydai) [reply src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py]: https://www.udacity.com/wiki/cs258/truthiness-in-python

File: src/command_modules/azure-cli-rdbms/setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.2.3"
+VERSION = "0.2.4"
```
Comment (troydai) [inline src/command_modules/azure-cli-rdbms/setup.py]: Additional question (non-blocking). Is the service GA now? If so, when will the version be bumped to `1.x.x`

Comment (tjprescott) [reply src/command_modules/azure-cli-rdbms/setup.py]: GA modules in the CLI start at version 2.0.0.

File: src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py (hunk 1 above)
Comment (ravindrakreddy) [reply src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py]: fixed

File: src/command_modules/azure-cli-rdbms/setup.py (hunk 3 above)
Comment (ravindrakreddy) [reply src/command_modules/azure-cli-rdbms/setup.py]: I will work with our PM's to check if we need to bump the version to 1.0 or 2.0 as service is GA now. This is not applicable for this PR

**Comment by troydai** [comment]
There is still `Do not merge` in the subject line. Is this change ready for merge?

**Comment by ravindrakreddy** [comment]
@troydai : Changed the title.  Please make sure this goes live in next cli release

**Pull request merged**